src/libs/functions/formatRedirectLink.ts

```diff
@@ -3,7 +3,7 @@ import CozyClient, {
   deconstructRedirectLink
 } from 'cozy-client'
 
-export const formatOnboardingRedirection = (
+export const formatRedirectLink = (
```

Comment:
should be put in cozy-client?

Reply:
Second times we used this function, second person that talk about putting it in cozy-client => let's go